### PR TITLE
Parse Args from terminal

### PR DESCRIPTION
**Pull request by Fil8**
RFInder now accepts most of the options of the config file from terminal.

- `rfinder -h` provides an help
- `rfinder -c config_file.yml` allows to run with preset config file
- `rfinder` reads the default config file (or a file called rfinder_default.yml, if it exists in your current directory)
- `rfinder -i MSFILE -tel telescope` allows to specify filename in the current directory 

other options are available through the help.

A log file is created in log-rfinder.log. This logfile is overwritten each time rfinder is called.

### CODE REVIEW

File: rfinder/__init__.py [hunk 1]
```diff
@@ -362,14 +520,86 @@ def go(self,cfg_par):
...
+        elif args.config:    #rfinder -c config_file.yml         
+            self.logger.warning('------ Reading your parameter file ------\n')
+            # read database here
+            files =  args.config
+            cfg = open(files)
+            self.cfg_par = yaml.load(cfg)
+
+        else: #rfinder  or rfinder -options
+            workdir = os.getcwd()
+            workdir = workdir+'/'
```
Comment (Athanaseus) [inline rfinder/__init__.py]: I think you don't really need this line since below you do `workdir+'/'+DEFAULT_CONFIG`

**Comment by Athanaseus** [comment]
About the readme checkout: https://stackoverflow.com/questions/26737222/pypi-description-markdown-doesnt-work, basically there are two formats; markdown (.md) and reStructured Text(.rst), easier just to port to `.rst` which I believe is automatically rendered or otherwise try specifying key `long_description_content_type='text/markdown',` in the `setup.py` file

File: setup.py [hunk 2]
```diff
@@ -18,12 +18,13 @@ def requirements():
...
         return [pname.strip() for pname in f.readlines()]
 
-__version__ = "1.0.2"
+__version__ = "1.0.3"
 
 setup(name=pkg,
       version=__version__,
       description="A tool to identify the presence of RFI in an observation and visualize it",
       long_description=readme(),
+      long_description_content_type = 'text/markdown'
```
Comment (Athanaseus) [inline setup.py]: missing a comma here

Comment (Fil8) [reply setup.py]: :+1:

File: rfinder/__init__.py (hunk 1 above)
Comment (Fil8) [reply rfinder/__init__.py]: done

**Pull request merged**